highlightCode.js

```diff
@@ -1,5 +1,19 @@
 const CodeMirror = require("codemirror/addon/runmode/runmode.node");
 
+const tagsToReplace = {
+   '&': '&amp;',
+   '<': '&lt;',
+   '>': '&gt;'
+};
+
+const replaceTag = function(tag) {
+   return tagsToReplace[tag] || tag;
+}
+
+const safe_tags_replace = function(str) {
```

Comment:
could you please make the naming consistent? The rest of the declarations use camelCase, while this one switches to snake case for some reason.

Reply:
Thanks. I renamed the snake case function (originating from stack overflow) to camel case and renamed the functions to be more descriptive.